core/reporter-api/src/main/java/org/arquillian/reporter/api/model/entry/FileEntry.java

```diff
@@ -1,9 +1,40 @@
 package org.arquillian.reporter.api.model.entry;
 
+import java.nio.file.Path;
+
 /**
  * An {@link Entry} representing a file
  *
  * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
  */
 public class FileEntry implements Entry {
```

Comment:
Then maybe we can left as FileEntry, being the base type an String but a constructor receiving a `Path` or a `File`. Also we can add a new property that is the type of file, (in free string) where user can set if it is a video, image, ...